components/utils/Card.tsx

```diff
@@ -22,14 +22,14 @@ export default function Card({
 
 const styles = StyleSheet.create({
   card: {
-    marginHorizontal: 8,
-    borderRadius: 18,
-    paddingTop: 12,
-    paddingBottom: 12,
+    margin:10,
+    // marginHorizontal: 8,
+    // paddingTop: 12,
+    // paddingBottom: 12,
+    borderRadius: 29,

```

Comment:
if we remove this it affects the RequestCard UI:
![image](https://github.com/user-attachments/assets/35bfd99c-e89b-44d4-85f0-3168f825ee3e)